packages/framework/src/Component/Environment/EnvironmentFileSetting.php

```diff
@@ -8,10 +8,21 @@ class EnvironmentFileSetting
         EnvironmentType::DEVELOPMENT => 'DEVELOPMENT',
         EnvironmentType::PRODUCTION => 'PRODUCTION',
         EnvironmentType::TEST => 'TEST',
+        EnvironmentType::GOOGLE_CLOUD => 'GOOGLE_CLOUD',
```

Comment:
I don't think an extra environment is a good solution to this. In my opinion, setting up the GCS bucket as a file system adapter should be orthogonal - i.e. it should be independent of the other settings. It shouldn't be impossible to run a dev, test or production environment with files stored in GCS.

Eg. an ENV variable GOOGLE_CLOUD_STORAGE_BUCKET_NAME could be introduced. If empty, the application would behave just as it behaves right now, otherwise it would use the GCS adapter. This should work in all three environments.

Can we please discuss the configuration options futher?